tedge/src/cli/disconnect/bridge.rs

```diff
@@ -0,0 +1,107 @@
+use crate::cli::connect::TEDGE_BRIDGE_CONF_DIR_PATH;
```

Comment:
I would not automatically expect that `DisconnectBridge` is defined in file `bridge.rs`.
You could name it `command.rs` or `disconnect_bridge.rs`....

Reply:
Thanks for the names. I was hoping someone could come with better names :D